libs/server/auth/shell/src/lib/server-auth-shell.module.ts

```diff
@@ -17,9 +29,11 @@ const providers = [{ provide: BaseAuthRepository, useClass: AuthRepository }];
         secret: configService.getJwtSecret()
       })
     }),
-    ServerAuthCoreApplicationServicesModule
+    ServerAuthInfrastructureModule,
+    ServerAuthCoreApplicationServicesModule,
+    ServerUserShellModule
   ],
   providers,
-  exports: [...providers, ServerAuthCoreApplicationServicesModule, JwtModule]
+  exports: [...providers, ServerUserShellModule, ServerAuthCoreApplicationServicesModule, JwtModule]
```

Comment:
Should all of the elements be moved to the provider's array as it's done in server-auth-infrastructure.module.ts?